reference/docs-conceptual/whats-new/Migrating-from-Windows-PowerShell-51-to-PowerShell-7.md

```diff
@@ -0,0 +1,326 @@
+> and run the `Enable-SSH` cmdlet.
+
+The `New-PSSession`, `Enter-PSSession`, and `Invoke-Command` cmdlets have new parameter sets to
+support SSH connections.
+
+```powershell
+[-HostName <string>]  [-UserName <string>]  [-KeyFilePath <string>]
+```
+
+To create a remote session, specify the target computer with the **HostName** parameter and provide
+the user name with **UserName**. When running the cmdlets interactively, you're prompted for a
+password.
+
+```powershell
+Enter-PSSession -HostName <Computer> -UserName <Username>
+```
+
+Alternatively, when using the **HostName** parameter, provide the username information followed by
+the at sign ('@'), followed by the computer name.
+
+```powershell
+Enter-PSSession -HostName <Username>@<Computer>
+```
+
+You may set up SSH key authentication using a private key file with the **KeyFilePath** parameter.
+For more information, see [OpenSSH Key Management](/windows-server/administration/openssh/openssh_keymanagement).
+
+### Group Policy supported
+
+PowerShell includes Group Policy settings to help you define consistent option values for servers in
+an enterprise environment. These settings include:
+
+- Console session configuration: Sets a configuration endpoint in which PowerShell is run.
+- Turn on Module Logging: Sets the LogPipelineExecutionDetails property of modules.
+- Turn on PowerShell Script Block Logging: Enables detailed logging of all PowerShell scripts.
+- Turn on Script Execution: Sets the PowerShell execution policy.
+- Turn on PowerShell Transcription: enables capturing of input and output of PowerShell commands
+  into text-based transcripts.
+- Set the default source path for Update-Help: Sets the source for Updatable Help to a directory,
+  not the Internet.
+
+For more information, see
+[about_Group_Policy_Settings](/powershell/module/microsoft.powershell.core/about/about_group_policy_settings).
+
+PowerShell 7 includes Group Policy templates and an installation script in `$PSHOME`.
+
+Group Policy tools use administrative template files (`.admx`, `.adml`) to populate policy settings
+in the user interface. This allows administrators to manage registry-based policy settings. The
+`InstallPSCorePolicyDefinitions.ps1` script installs PowerShell Core Administrative Templates on the
+local machine.
+
+```powershell
+Get-ChildItem -Path $PSHOME -Filter *Core*Policy*
+```
+
+```Output
+    Directory: C:\Program Files\PowerShell\7
+
+Mode                 LastWriteTime         Length Name
+----                 -------------         ------ ----
+-a---           2/27/2020 12:38 AM          15861 InstallPSCorePolicyDefinitions.ps1
+-a---           2/27/2020 12:28 AM           9675 PowerShellCoreExecutionPolicy.adml
+-a---           2/27/2020 12:28 AM           6201 PowerShellCoreExecutionPolicy.admx
+```
+
+### Separate Event Logs
+
+PowerShell 7 includes separate event logs for Windows PowerShell and PowerShell Core. Use the
```

Comment:
Unless we're actually forwarding the events from Windows PS to the PS7 event logs (which I don't think we are), I think this is more accurate: 

```suggestion
Windows PowerShell and PowerShell 7 log events to separate event logs. Use the
```